packages/devextreme/js/localization/messages/es.json

```diff
@@ -417,6 +417,7 @@
         "dxHtmlEditor-delete": "Delete",
         "dxHtmlEditor-border": "Border",
         "dxHtmlEditor-style": "Style",
+        "dxHtmlEditor-borderWidth": "Border Width",
```

Comment:
[for the future, no need to change now]
It's better to locate new variables in a "sorted" order. For example below we have "dxHtmlEditor-borderColor" variable => I expect new "dxHtmlEditor-borderWidth" variable is located just after this one

Reply:
[ec57a58](https://github.com/DevExpress/DevExtreme/pull/27994/commits/ec57a5870ae3b9e9bd13f892e39dfebb892e289b)